app/src/ui/pull-request-quick-view.tsx

```diff
@@ -121,6 +125,40 @@ export class PullRequestQuickView extends React.Component<
     return { top: clamp(alignedMiddle, minTop, maxTop) }
   }
 
+  private getPointerPosition(
+    position: React.CSSProperties | undefined,
+    quickViewHeight: number
+  ): React.CSSProperties | undefined {
+    // 23 = half a pr list item.
+    const defaultTop = { top: 20 }
+    if (position === undefined) {
+      return defaultTop
+    }
+
+    const { top, bottom } = position
+    const quickViewTopZero = this.getTopPRList()
+    const prListItemTopWRTQuickViewTopZero =
+      this.props.pullRequestItemTop - quickViewTopZero
+
+    let quickViewTop
+    if (top !== undefined) {
+      quickViewTop = parseInt(top.toString())
```

Comment:
The fact that we have to coerce this to a string and then parse it suggests to me that we're not leveraging the storage type properly. Looking at `state.position` it's a `CSSProperties` type but it seems we only ever use it to set the `top` property?

Could we change the state type to something like

```ts
interface IPullRequestQuickViewState {
  /** The absolutely positioned top of the quickview element in pixels  */
  readonly top: number | undefined
}
```